source/tunneling/SecureTunnelingContext.cpp

```diff
@@ -22,14 +22,14 @@ namespace Aws
 
                 SecureTunnelingContext::SecureTunnelingContext(
                     shared_ptr<SharedCrtResourceManager> manager,
-                    const string &rootCa,
+                    Aws::Crt::Optional<std::string> &rootCa,
                     const string &accessToken,
                     const string &endpoint,
                     uint16_t port,
                     OnConnectionShutdownFn onConnectionShutdown)
                 {
                     mSharedCrtResourceManager = manager;
```

Comment:
Two recommendations.
* Make it a const reference eg `const Aws:...`
* Replace assignment in constructor body with use of member initialization list eg `: mSharedCrtResourceManager(manager), mRootCa(rootCa.has_value() ? rootCa.value() : basic_string<char>("")), ...`